packages/sdk/base/src/identifier.ts

```diff
@@ -0,0 +1,53 @@
+export const PEPPER_SEPARATOR = '__'
+
+/**
+ * Standardized prefixes for ODIS identifiers.
+ * Used in @celo/identity, @celo/phone-utils, and the protocol package.
+ *
+ * When adding prefixes, make sure to add the expected value for the unit test case
+ * in identifier.test.ts, otherwise the test will fail.
+ */
+export enum IdentifierPrefix {
```

Comment:
why did we move this to the base package?

Reply:
I did this to maintain the structure of these functions & minimize duplication; `getIdentifierHash` -> `getPrefixedIdentifier` -> `IdentifierPrefix`, and `phone-utils` also needs the `IdentifierPrefix.PHONE_NUMBER`. I laid out a couple other alternatives that are possible in the PR description now though, lmk what you think + happy to discuss in more detail!